types/src/system/auction/error.rs

```diff
@@ -262,6 +263,13 @@ pub enum Error {
     /// assert_eq!(39, Error::ArithmeticOverflow as u8);
     /// ```
     ArithmeticOverflow = 39,
+    /// An error that is raised when there is an error in the mint contract that cannot
+    /// be mapped to a specific auction error.
+    /// ```
+    /// # use casper_types::system::auction::Error;
+    ///  assert_eq!(40, Error::MintError as u8);
+    /// ```
+    MintError = 40,
```

Comment:
We should move GasLimit up and assign it a number. We did something similar for other system contract recently iirc

```suggestion
    #[doc(hidden)]
    GasLimit = 40,
    MintError = 41,
```

Reply:
I agree with @mpapierski - this is a blocker as otherwise we're breaking the API.

Reply:
Applied @mpapierski suggestion